### DISCUSSION TITLE
Feat: `cw wrapped`

### DISCUSSION THREAD

**Discussion by 5ouma** [aws/codewhisperer-command-line-discussions#29] [Feature request]
### Checks

- [X] I could not find an existing setting that solved my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/codewhisperer-command-line-discussions/discussions?q=) and there are no duplicates of my feature request

### Feature Details

Fig has a unique feature, `fig wrapped`, to show analytics data.
<details><summary>🎁 <a href="https://twitter.com/fig/status/1470512802431311872">Run `fig wrapped` to see how you used the terminal in 2021.</a></summary>
<img src="https://pbs.twimg.com/media/FGhPJYuUUAIKfe5?format=jpg" />
</details>
I like this feature, and CodeWhisperer for Command Line should have too!

**Comment by chaynabors** [comment]
Closing in favor of https://github.com/aws/codewhisperer-command-line-discussions/discussions/42
